chromium_src/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc

```diff
@@ -108,4 +118,36 @@ std::u16string ChromeAutocompleteProviderClient::GetClipboardText() const {
 #endif
 }
 
-#endif  // BUILDFLAG(ENABLE_AI_CHAT)
+bool ChromeAutocompleteProviderClient::IsQuerySafeToSearchSuggestions(
```

Comment:
why are we adding this method to `ChromeAutocompleteProviderClient`? It doesn't appear to require an instance of `ChromeAutocompleteProviderClient`

Reply:
Deleted this interface by moving check logic to `brave/components/omnibox`